extensions/wikidata/src/org/openrefine/wikidata/commands/PreviewResults.java

```diff
@@ -4,7 +4,7 @@
 
 import org.openrefine.wikidata.qa.QAWarning;
 import org.openrefine.wikidata.qa.QAWarning.Severity;
-import org.openrefine.wikidata.updates.TermedStatementEntityUpdate;
+import org.openrefine.wikidata.updates.TermedStatementEntityEdit;
```

Comment:
@j-sal in this PR I am renaming (again!) this class to avoid name clashes between our classes (now `Edit`s, before `Update`s) and Wikidata-Toolkit's clases (still `Update`s).
I will try to draw a diagram to explain the relationship between all those layers, because the distinctions are subtle!

Reply:
Here is the diagram: https://deploy-preview-4520--openrefine-docs.netlify.app/next/technical-reference/wikibase/architecture